app/src/ActionModal/components/DelegationModal.vue

```diff
@@ -213,6 +222,7 @@ export default {
     delegations: [],
     messageType,
     smallestAmount: SMALLEST,
+    isInElection: false, // Handle election period in Polkadot
```

Comment:
this needs to be reset on a close?

Reply:
it will just update automatically according to the current block. Why do we need to reset it?

Reply:
Yep, we dont need to.